docs/prob/_sidebar.md

```diff
@@ -20,5 +20,20 @@
   - [/prob/api/unexpected-db-write-error](/prob/api/unexpected-db-write-error.md)
   - [/prob/api/unexpected-multipart-read-error](/prob/api/unexpected-multipart-read-error.md)
   - [/prob/build/run/invalid-input](/prob/build/run/invalid-input.md)
+  - [/prob/provider/azuredevops/building-url-error](/prob/provider/azuredevops/building-url-error.md)
+  - [/prob/provider/azuredevops/constructing-git-url-error](/prob/provider/azuredevops/constructing-git-url-error.md)
+  - [/prob/provider/azuredevops/empty-group-error](/prob/provider/azuredevops/empty-group-error.md)
+  - [/prob/provider/azuredevops/empty-user-error](/prob/provider/azuredevops/empty-user-error.md)
+  - [/prob/provider/azuredevops/getting-body-error](/prob/provider/azuredevops/getting-body-error.md)
+  - [/prob/provider/azuredevops/getting-build-definition-error](/prob/provider/azuredevops/getting-build-definition-error.md)
+  - [/prob/provider/azuredevops/getting-provider-error](/prob/provider/azuredevops/getting-provider-error.md)
+  - [/prob/provider/azuredevops/getting-token-error](/prob/provider/azuredevops/getting-token-error.md)
+  - [/prob/provider/azuredevops/invalid-event-type-error](/prob/provider/azuredevops/invalid-event-type-error.md)
+  - [/prob/provider/azuredevops/posting-provider-error](/prob/provider/azuredevops/posting-provider-error.md)
+  - [/prob/provider/azuredevops/posting-token-error](/prob/provider/azuredevops/posting-token-error.md)
+  - [/prob/provider/azuredevops/putting-branch-error](/prob/provider/azuredevops/putting-branch-error.md)
+  - [/prob/provider/azuredevops/putting-project-error](/prob/provider/azuredevops/putting-project-error.md)
+  - [/prob/provider/azuredevops/send-trigger-error](/prob/provider/azuredevops/send-trigger-error.md)
+  - [/prob/provider/azuredevops/unmarshal-project-error](/prob/provider/azuredevops/unmarshal-project-error.md)
```

Comment:
There are a lot of problems here. Maybe you could generalize some of them? Maybe it's not that important to have 4 different problem types for posting/putting project data into the Wharf API